backend/api/Services/InstallationService.cs

```diff
@@ -137,6 +144,33 @@ public async Task<Installation> Update(Installation installation)
             return installation;
         }
 
+        public async Task EnsureRobotIsOnSameInstallationAsMission(
```

Comment:
Can we name it AssertRobotIs...?

Ensure sounds to me like we are changing it to become right if it is not right already
```suggestion
        public async Task AssertRobotIsOnSameInstallationAsMission(
```

Reply:
`context.Database.EnsureCreated();`
from https://github.com/equinor/flotilla/blob/a0a603f0c4e4f07a78317b7b6a4000e2ca573344/backend/api/Configurations/CustomServiceConfigurations.cs#L44

Is one example where ensure makes sense. From its documentation is says:
```
Ensures that the database for the context exists.

• If the database exists and has any tables, then no action is taken. Nothing is done to ensure the database schema is compatible with the Entity Framework model.
• If the database exists but does not have any tables, then the Entity Framework model is used to create the database schema.
• If the database does not exist, then the database is created and the Entity Framework model is used to create the database schema.
```

In other words, if the condition is not met; actions will be taken. 

Since that is not the case in our method here, I think we should refrain from using "ensure" in the name

Reply:
I agree I just moved the function from one service to another, but I will rename it as it fits better to what it actually does